connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -246,3 +248,20 @@ message SubqueryAlias {
   // Optional. Qualifier of the alias.
   repeated string qualifier = 3;
 }
+
+// StatFunction
+message StatFunction {
+  // Required. The input relation.
+  Relation input = 1;
+  // Required. The function and its parameters.
+  oneof function {
+    Summary summary = 2;
+
+    Unknown unknown = 999;
```

Comment:
why do we need this?

Reply:
here follows https://github.com/apache/spark/blob/master/connector/connect/src/main/protobuf/spark/connect/relations.proto#L51 to catch unexpected input

Reply:
I think that's for enum but here is an optional field... cc @amaliujia

Reply:
Question: will we add new functions under this `oneof`?

Reply:
yes, such as `crosstab` `cov` `corr` etc

Reply:
ok then this makes sense